docs/mapping/freiburg.md

```diff
@@ -0,0 +1,74 @@
+# Realtime and Static data of the city of Freiburg
+
+The city of Freiburg provides realtime ``GEOJSON`` parking data and also static nd realtime Park + Ride data for cars.
+
+## ParkingSites
+
+A `ParkingSites` provides realtime data for a `ParkingSite`.
+
+| Field                      | Type                     | Cardinality | Mapping                         | Comment                                                             |
+|----------------------------|--------------------------|-------------|---------------------------------|---------------------------------------------------------------------|
+| obs_parkid                 | integer                  | 1           | uid                             |                                                                     |
+| obs_max                    | integer                  | ?           | realtime_capacity/capacity      |                                                                     |
+| obs_free                   | integer                  | ?           | realtime_free_capacity          |                                                                     |
+| obs_ts                     | datetime                 | 1           | realtime_data_updated_at        |                                                                     |
+| obs_state                  | integer                  | ?           | #RealtimeOpeningStatus          |                                                                     |
+| public_url                 | string                   | ?           | public_url                      |                                                                     |
+
+
+## ParkingSites for P+R Static
+
+Attributes which are set statically:
+* `has_realtime_data` is always set to `False`
+
+A `ParkingSites` provides static data for a Park and Ride `ParkingSite`.
+
+| Field                      | Type                     | Cardinality | Mapping                         | Comment                                                             |
+|----------------------------|--------------------------|-------------|---------------------------------|---------------------------------------------------------------------|
+| ogc_fid                    | integer                  | 1           | uid                             |                                                                     |
+| kapazitaet                 | integer                  | 1           | capacity                        |                                                                     |
+| name                       | string                   | 1           | name                            |                                                                     |
+| nummer                     | string                   | ?           | name                            |                                                                     |
```

Comment:
`cardinality` is actually 1, as the field always exists and is always the correct type string, it's just emptystring. You can add that to the comment section that you will ignore emptystring.